docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+# Create an NFT Collection Using LSP8
+
+This tutorial goes over creating a collection of collectibles, where each item is unique and has its own design.
+
+:::info
+⌨️ The full code of this example can be found in 👾 [here](https://github.com/CJ42/LUKSO-Hardhat-template).
+:::
+
+## Setup
+
+Make sure you have the following dependencies installed:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) (alternatively you can use [`web3.js`](https://github.com/web3/web3.js))
+- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+
+## Create the Smart Contracts
+
+When creating digital asset smart contracts on LUKSO, you will need to specify the token type and the following data keys in the ERC725Y storage. There are three types that you can define:
+
+- 0 = Token
+- 1 = NFT
+- 2 = Collection
+
+```solidity title="contracts/TokenTypes.sol"
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+bytes32 constant _LSP4_TOKEN_TYPE_DATA_KEY = 0xe0261fa95db2eb3b5439bd033cda66d56b96f92f243a8228fd87550ed7bdfdb3;
+
+enum TokenType {
+    TOKEN,
+    NFT,
+    COLLECTION
+}
+
+
+```
+
+You will also create a custom [LSP8 Identfiable Digital Asset Collection](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) that extends [LSP8Mintable](../../contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.md).
+
+```solidity title="contracts/MyNFTCollection.sol"
+
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+// modules
+import {
+    LSP8Mintable
+} from "@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.sol";
+
+// constants
+import {
+    _LSP8_TOKENID_TYPE_NUMBER
+} from "@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol";
+import {_LSP4_TOKEN_TYPE_DATA_KEY, TokenType} from "../TokenTypes.sol";
+
+contract BasicNFTCollection is LSP8Mintable {
+    constructor(
+        string memory nftCollectionName,
+        string memory nftCollectionSymbol,
+        address contractOwner
+    )
+        LSP8Mintable(
+            nftCollectionName,        // NFT collection name
+            nftCollectionSymbol,      // NFT collection symbol
+            contractOwner,            // owner of the NFT contract (the address that controls it, sets metadata, can transfer the ownership of the contract)
+            _LSP8_TOKENID_TYPE_NUMBER // type of NFT/ tokenIds
+        )
+    {
+        // set token type
+        _setData(_LSP4_TOKEN_TYPE_DATA_KEY, abi.encode(TokenType.COLLECTION));
+    }
+}
+```
+
+## Deploy our NFT Collection Contract on the LUKSO Testnet
+
```

Comment:
I would suggest adding a sentence here to introduce the people into the next section, maybe even put a "info" box for hardhat for which network it can be deployed.

Explain why `import {BasicNFTCollection, BasicNFTCollection__factory} from "../typechain-types";` is needed and where users are able to specify their network for the deployment (maybe just a notice: `it will automatically use your default network setup`)